src/renderer/components/layout/BreadCrumb.tsx

```diff
@@ -45,24 +45,26 @@ class BreadCrumb extends React.Component<IProps> {
                         <SVG svg={ArrowIcon} />
                     </Link>
                 }
-                {breadcrumb && breadcrumb.map((item, index) =>
-                    item.path && index !== breadcrumb.length - 1 ?
+                {breadcrumb && breadcrumb.map((item, index) => {
+                    const name = decodeURI(item.name);
+                    return (item.path && index !== breadcrumb.length - 1 ?
                         <Link
                             key={index}
                             to={{
                                 pathname: item.path,
                                 search: `?displayType=${search.displayType}`,
                                 state: item.state,
                             }}
-                            title={item.name}
+                            title={name}
                         >
-                            {`${decodeURI(item.name)} /`}
```

Comment:
did you forget to remove the second decodeURI ?
sorry if i'm wrong, i just look you code

Reply:
Oh yes...I checked it twice but apparently the morning is a difficult period. Thank you

Reply:
Ah, no it's ok, I did'nt forgot anything